content/faq/security/general.md

```diff
@@ -26,9 +26,11 @@ You can configure this through SSO using your IdP. Check with your IdP if they s
 
 ### How are sessions managed and do they expire?
 
-If configured, the IdP manages sessions.
+Docker Desktop uses tokens to manage sessions after a user signs in. Docker Desktop signs you out after 90 days, or 30 days of inactivity.
 
-Docker Desktop sessions expire after 30 days, or after 7 days of inactivity. For Docker Hub, the IdP manages sessions, if configured. If you use application-level sign-in, users are signed out due to inactivity after 14 days and must sign in again after 30 days.
+In Docker Hub, you need to re-authenticate after 24 hours. If users are authenticating using SSO, the default session timeout for the IdP is respected.
```

Comment:
**Note for technical reviewers:** Still not 100% clear on if the second line here about SSO is accurate. Can remove if needed!

Reply:
Checked in the docs for IdPs like Okta and Azure, like Ivan mentioned in our meeting this is usually around 1 hour so less than the timeout we use. Resolving.